lib/src/model/common/eval.dart

```diff
@@ -71,10 +82,11 @@ class ClientEval with _$ClientEval implements Eval {
     return Move.fromUci(uci);
   }
 
-  IList<Move?> get bestMoves {
+  IList<MoveWithWinningChanges> bestMoves() {
```

Comment:
Why remove the getter here and use a method instead?

Reply:
That happened by accident, fixed now.